src/bp/core/migration/migration-service.test.ts

```diff
@@ -0,0 +1,71 @@
+import 'reflect-metadata'
+import 'bluebird-global'
+
+import { GhostService } from 'core/bpfs'
+import { ConfigProvider } from 'core/config'
+import Database from 'core/database'
+import { PersistedConsoleLogger } from 'core/logger'
+import { createSpyObject, MockObject } from 'core/misc/utils'
+import yn from 'yn'
+import { MigrationService } from './migration-service'
+
+// TODO: Improve these unit tests and add more of them
```

Comment:
How could they be improved?

Reply:
One example would be to make sure debug is called in the first test, but since mocking everything related to global.DEBUG is harder than I thought, I opted for the sorter path.